### PR TITLE
Allow for local images to be embedded in report

### PR DESCRIPTION
**Pull request by maggiesavovska**
The current functionality assumes that when a user adds an image via the addContext method, the image added is always accessed via url (non local path). The changes allow the user to add a path to an image contained on the local file system to be embedded into the report.

### CODE REVIEW

**Comment by maggiesavovska** [comment]
Changed starting width of the local image within the additional context section to 30% width and added ability to expand the image to 100% width on click. This reduces the vertical space the image takes up in the report so that the report can be read more efficiently (assuming that the image would most likely be a screenshot of a page).

File: src/components/test/context.jsx [hunk 1]
```diff
@@ -52,28 +89,28 @@ class TestContext extends Component {
...
+    }
+
+    if (!rawContent) {
+      return false;
+    }
+
+    if (urlRegEx.test(rawContent)) {
+      addContent = this.renderLink(rawContent, title);
+    } else {
+      addContent = imgRegEx.test(rawContent) ? this.renderImage(rawContent, title) :
```
Comment (AlehVasilyeu) [inline src/components/test/context.jsx]: I guess it better to place "?" and ":" on own lines, it makes it more readable. :)

**Comment by adamgruber** [comment]
@maggiesavovska Take a look at #6 . I'm still not sure about having the image enlarge on click. As of now if you click the image it will load it in a new window so the enlarge feature would overwrite that behavior. It could be that we need more UX around that, like a hover state on the image with options to zoom or open in a new window but since that's beyond the scope of this issue I figure it can be handled in a future PR.

**Comment by maggiesavovska** [comment]
@adamgruber: I took a look and it looks good. Yes, it would be great in a future PR to solve the issue of the image taking up too much space. I need my users to get a quick overview of the screenshots because the testing I'm doing is very much visually oriented. But this solution looks great. Thanks!

**Comment by adamgruber** [comment]
1.0.4 Released

**Comment by chandramuralis** [comment]
Is there any example on how to embed image inline?

**Pull request closed without merging**